labs/aws/python/lab-05/README.md

```diff
@@ -38,7 +38,8 @@ Next, we need to declare a new Kubernetes provider based on the kubeconfig creat
 the cluster stack. To do this, add this to your `__main__.py` file
 
 ```python
-k8s_provider = Provider("k8s-provider", kubeconfig=infra.get_output("kubeconfig"))
+kubeconfig = infra.get_output("kubeconfig").apply(lambda c: json.dumps(c))
```

Comment:
I needed this because I was using TypeScript for the cluster, and Python for the apps.  I think we should likely be consistent across languages in whether the export is a string or a JSON object.  It may be we want to fix this the other direction by making it a string everywhere?  But it is slightly nicer to get JSON output in the stack output.

Aside - we should really allow an object or string as the input to `kubeconfig` on a `k8s.Provider`.

Reply:
> But it is slightly nicer to get JSON output in the stack output.

Agreed that an object is better for stack output. This is tracked in https://github.com/pulumi/pulumi-eks/issues/359 but I am hesitant to change the shape of the kubeconfig here for existing clusters.

> Aside - we should really allow an object or string as the input to kubeconfig on a k8s.Provider.

The k8s provider seems like the right spot to support both of these cases. This is being tracked in https://github.com/pulumi/pulumi-kubernetes/issues/1032

Reply:
If an object is better (which I agree it is) we don’t need to change the TS EKS package - we just have to change the Go and Python cluster labs to export an object (and then take a change like the one I propose here for Go as well). Right?

Reply:
> we just have to change the Go and Python cluster labs to export an object (and then take a change like the one I propose here for Go as well). Right?

Yup, I'll get a PR going to fix this